org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -462,7 +510,10 @@ protected override void OnHoverEntered(HoverEnterEventArgs args)
                 }
 
                 validPokeInteractors.Add(pokeInteractor);
+                isHovered = true;
             }
+
+            ProximityHoverEntered?.Invoke(this, null, args);
```

Comment:
See my comment with the `UpdateProximityHovered()` suggestion....you should remove the event invoking here and call `UpdateProximityHovered()`

Reply:
Implemented in latest commit.  Resolving this.